src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
+  rejectedQuotes: "Unexpected quotes",
+})
+
+function isQuotesExist(str) {
```

Comment:
Oh.

@evilebottnawi Are we sure whitespace is allowed around the operator and inside the brackets? If so, we should probably fire off a PR upsteam to `postcss-selector-parser` to account for this (an amendment this [this PR](https://github.com/postcss/postcss-selector-parser/pull/59/files).